tests/common/mod.rs

```diff
@@ -1,13 +1,24 @@
 #![cfg_attr(feature = "aggressive_lint", deny(warnings))]
 
-use rusoto_core::Region;
-use rusoto_s3::S3Client;
+// Not all test modules use everything in this helper module
```

Comment:
You can get around this by structuring the integration tests like this - https://joshleeb.com/blog/rust-integration-tests/. It makes all the test belong to the same crate rather than a crate for each test file. I tried it [here](https://github.com/swift-nav/esthri/pull/65) and it seems to work. Not sure if it's actually better, just an alternative idea

Reply:
If it gets ride of the dead_code annotation, then I'd say it's better, I don't think we really need the separate binaries.  As long as we can still do things like running one particular test (especially under a debugger) then I don't see a downside.  I had to run the download tests under a debugger to see why it was hanging, and it was convenient to just run that one particular integration test.

Reply:
Ya you can still run a specific test with `cargo test <name or prefix of test>` so that should be alright